pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -345,20 +322,15 @@ func (og *operationGenerator) GenerateDetachVolumeFunc(
 		}
```

Comment:
@saad-ali & @tsmetana - I went ahead and added an error for this particular situation.